haskell/haskell.bzl

```diff
@@ -1,38 +1,124 @@
-HASKELL_FILETYPE = FileType([".hs"])
+# TODO: using run_shell doesn't work, some wrong nix store path to
```

Comment:
This might be fixed in latest Nixpkgs. Did you try that? I haven't encountered anything similar (I'm also on NixOS).